dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -381,6 +409,7 @@ pub fn invoke_export(resource: &ResourceManifest, cwd: &str) -> Result<ExportRes
 /// Error is returned if the command fails to execute or stdin/stdout/stderr cannot be opened.
 #[allow(clippy::implicit_hasher)]
 pub fn invoke_command(executable: &str, args: Option<Vec<String>>, input: Option<&str>, cwd: Option<&str>, env: Option<HashMap<String, String>>) -> Result<(i32, String, String), DscError> {
+    debug!("Invoking command {} with args {:?}", executable, args);
```

Comment:
what is your logic when to use `debug!` or `info!` for diagnostics?

Reply:
`info` for me is going to be used for `verbose` which can be useful to end-users while `debug` is really only useful to developers of `dsc` itself

Reply:
ok; let's stick with this approach.